dataland-frontend/src/components/resources/dataTable/MultiLayerDataTable.vue

```diff
@@ -55,86 +55,64 @@
-      let referencedReports;
-      switch (singleDataAndMetaInfo.metaInfo.dataType) {
-        case "sfdr":
-          referencedReports = (singleDataAndMetaInfo.data as SfdrData).general?.general.referencedReports;
-          break;
-        case "eutaxonomy-financials":
-          referencedReports = (singleDataAndMetaInfo.data as EuTaxonomyDataForFinancials).referencedReports;
-          break;
-        case "eutaxonomy-non-financials":
-          referencedReports = (singleDataAndMetaInfo.data as EutaxonomyNonFinancialsData).general?.referencedReports;
-          break;
-        default:
-          referencedReports = null;
-          break;
-      }
-      if (referencedReports) {
-        for (const key in referencedReports) {
-          const companyReport: CompanyReport | undefined = referencedReports[key];
-          const reportDate = companyReport?.reportDate;
-          if (reportDate && (!latestDate || reportDate > latestDate)) {
-            latestDate = reportDate;
-          }
-        }
+const vTooltip = Tooltip;
+
+/**
+ * Generates the toolTip for reportingYear given DataAndMetaInformation.
+ * @param singleDataAndMetaInfo the DataAndMetaInformation of a framework.
+ * @returns string the toolTip.
+ */
+function reportingYearToolTip(singleDataAndMetaInfo: DataAndMetaInformation<T>): string {
+  let latestDate = null;
+  let referencedReports;
+  switch (singleDataAndMetaInfo.metaInfo.dataType) {
+    case "sfdr":
+      referencedReports = (singleDataAndMetaInfo.data as SfdrData).general?.general.referencedReports;
+      break;
+    case "eutaxonomy-financials":
+      referencedReports = (singleDataAndMetaInfo.data as EuTaxonomyDataForFinancials).referencedReports;
+      break;
+    case "eutaxonomy-non-financials":
+      referencedReports = (singleDataAndMetaInfo.data as EutaxonomyNonFinancialsData).general?.referencedReports;
+      break;
+    default:
+      referencedReports = null;
+      break;
+  }
+  if (referencedReports) {
+    for (const key in referencedReports) {
+      const companyReport: CompanyReport | undefined = referencedReports[key];
+      const publicationDate = companyReport?.publicationDate;
+      if (publicationDate && (!latestDate || publicationDate > latestDate)) {
+        latestDate = publicationDate;
       }
-      const mostRecentSourceToolTip = latestDate ? `Publication date of most recent source:\n ${latestDate}\n\n` : "";
-      const datasetPublishedToolTip =
-        "Dataset published on Dataland:\n " +
-        convertUnixTimeInMsToDateString(singleDataAndMetaInfo.metaInfo.uploadTime);
-      return mostRecentSourceToolTip + datasetPublishedToolTip;
-    },
-  },
-});
+    }
+  }
+  const mostRecentSourceToolTip = latestDate
+    ? `Publication date of most recent report:\n ${dateStringFormatter(latestDate)}\n\n`
+    : "";
+  const datasetPublishedToolTip =
+    "Dataset published on Dataland on:\n " + convertUnixTimeInMsToDateString(singleDataAndMetaInfo.metaInfo.uploadTime);
```

Comment:
mit dem doppelt "on" finde ich sonderbar. Was meinst du zu "Publication date of the dataset on Dataland:"?